### PR TITLE
removed oadp prereq

### PR DESCRIPTION
**Pull request by dockerymick**
https://github.com/stolostron/backlog/issues/19104

### CODE REVIEW

File: clusters/backup_and_restore.adoc [hunk 1]
```diff
@@ -8,13 +8,7 @@ For a list of supported storage providers for Velero, see https://velero.io/docs
...
 == Prerequisites
 
-You must install the 0.4.2 version of the OADP Operator from the _beta_ channel. See the link:https://github.com/openshift/oadp-operator/blob/oadp-0.4.2/docs/install_olm.md[Install OADP Operator using OperatorHub] documentation. 
-
-Be sure to complete the steps to link:https://github.com/openshift/oadp-operator/blob/oadp-0.4.2/docs/install_olm.md#create-credentials-secret[Create credentials secret] for the cloud storage where the backups are saved. 
-
-Use the created secret when you link:https://github.com/openshift/oadp-operator/blob/oadp-0.4.2/docs/install_olm.md#create-the-velero-custom-resource[Create the Velero Custom Resource].
-
-**Note**: The cluster backup and restore operator resources must be created in the same namespace where the OADP Operator is installed.
+Are there any prereqs?
```
Comment (sahare) [inline clusters/backup_and_restore.adoc]: - You must install the latest version of the OADP Operator. See the link:https://github.com/openshift/oadp-operator/blob/master/docs/install_olm.md [Install OADP Operator using OperatorHub] documentation.

- Be sure to complete the steps to link:https://github.com/openshift/oadp-operator/blob/master/docs/install_olm.md#create-credentials-secret [Create credentials secret] for the cloud storage where the backups are saved. 

- Use the created secret when you link:https://github.com/openshift/oadp-operator/blob/master/docs/install_olm.md#create-the-dataprotectionapplication-custom-resource [Create the DataProtectionApplication Custom Resource].

 **Note**: The cluster backup and restore operator resources must be created in the same namespace where the OADP Operator is installed.

File: clusters/backup_restore_enable.adoc [hunk 2]
```diff
@@ -8,9 +8,7 @@ Enable the cluster backup and restore operator to schedule back up and restore f
 [#prereq-backup-restore-enable]
 == Prerequisites
 
-You must install the 0.4.2 version of the OADP Operator from the _beta_ channel. See the link:https://github.com/openshift/oadp-operator/blob/oadp-0.4.2/docs/install_olm.md[Install OADP Operator using OperatorHub] documentation.
-
-*Note*: Be sure to follow the OADP Operator installation instructions to create the Velero resource, and to create a valid connection to a backup storage location, where the backups are stored. View the link:https://github.com/openshift/oadp-operator/blob/oadp-0.4.2/docs/install_olm.md[Installation and setup steps] from the `oadp-operator` repository.
+Are there any prerequisites?
```
Comment (sahare) [inline clusters/backup_restore_enable.adoc]: - You must install the latest version of the OADP Operator. See the link:https://github.com/openshift/oadp-operator/blob/master/docs/install_olm.md [Install OADP Operator using OperatorHub] documentation.

*Note*: Be sure to follow the OADP Operator installation instructions to create the DataProtectionApplication resource, and to create a valid connection to a backup storage location, where the backups are stored. View the link:https://github.com/openshift/oadp-operator/blob/master/docs/install_olm.md [Installation and setup steps] from the `oadp-operator` repository.

**Review by sahare — CHANGES_REQUESTED** [review]
Thanks @dockerymick , please find the texts in the comments to replace the existing prereqs with.

**Comment by dockerymick** [comment]
@sahare thanks for your review and comments. I misunderstood the ask and assumed the requirement to install OADP should be removed. Will make updates soon, thanks!

**Pull request merged**